pkg/transportrequest/uploadcts.go

```diff
@@ -0,0 +1,227 @@
+package transportrequest
+
+import (
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"strings"
+)
+
+type fileUtils interface {
+	FileExists(string) (bool, error)
+}
+
+var files fileUtils = piperutils.Files{}
+
+// CTSConnection ...
```

Comment:
Maybe it would be helpful for review to add actual documentation, instead of this dummy docu.

Reply:
done